src/components/NewGalaxyModal.js

```diff
@@ -111,12 +155,68 @@ export const NewGalaxyModal = _.flow(
     ])
   }
 
+  const renderStoppingWarning = () => {
+    return h(Fragment, [
+      div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+        div([
+          ul({ style: { paddingLeft: '1rem', lineHeight: 1.5 } }, [
```

Comment:
The `ul` element represents a 'list' and should contain `li` elements. Since we're not conveying a list here, this content should probably just go directly in the `div`